rust_wtx_bench/.cargo/config.toml

```diff
@@ -0,0 +1,2 @@
+[build]
+rustflags = [ "-C", "target-cpu=native" ]
```

Comment:
Let's omit this flag. The reason it is not in other Rust implementations is that it:
1. May result in an illegal instruction segfault in some CPUs.
2. Image built on one machine may crash on another one.

There shouldn't be an assumption that the binary will be run on the exact same CPU where it was built. In practice, it's rarely the case.

Reply:
`grpcio`/`tonic` explicitly use this compiler flag and `wtx` performs manual vectorization when processing HPACK headers.

```rust
let mut iter = ArrayChunks::new(hpack_bytes);
for [a, b, c, d] in iter.by_ref() {
  // Process a...
  // Process b...
  // Process c...
  // Process d...
}
```

So `target-cpu=native` do have a great impact in the final generated assembly.

Trying a middle ground, I renamed everything from `native` to `x86-64-v3`. OK?

Reply:
To my understanding, the fact that libraries use a certain parameter doesn't matter much when using them in external projects. The flags that matter are defined in the target binary settings, e.g., here.

If we are to use this optimisation, it should be used across the board in all projects that have this option (all Rust ones, likely C/C++ as well). This can be potentially a follow-up PR.

Reply:
In https://github.com/memorysafety/zlib-rs/issues/123, a benchmark that should have been tied resulted in a performance difference of 80% due to the lack of `rustflags = [ "-C", "target-cpu=native" ]`. 

Rustc, as well as gcc/clang, defaults to a conservative set of x86 instructions and while the performance difference may not be as huge as the linked issue, some slowdown is likely going to occur.

```
$ rustc --print target-cpus

    x86-64                  - This is the default target CPU for the current build target (currently x86_64-unknown-linux-gnu).
    x86-64-v2
    x86-64-v3
    x86-64-v4
```

But yeah, I will just remove this parameter from all projects to make things more fair. Is that OK?

Reply:
Oh, it's my bad. I think I didn't check through the hidden directories it seems. Sorry for that!

Let's have it they way you had it originally, with the `native` in `wtx`.

Reply:
Sure, no problem!